src/components/RoiCalculatorModal/index.tsx

```diff
@@ -179,7 +179,7 @@ const RoiCalculatorModal: React.FC<RoiCalculatorModalProps> = ({
               $1000
             </Button>
             <Button
-              disabled={stakingTokenBalance.lte(0) || !account}
+              disabled={!stakingTokenBalance.isFinite() || stakingTokenBalance.lte(0) || !account}
```

Comment:
If stakingTokenBalance is NaN, lte condition will be false. Therefore button will be enabled